docs/jobs/openings/dev_rel.md

```diff
@@ -1,34 +1,39 @@
 ## Developer Relations opening at Aragon
 
-We are looking for a developer relations to help third party developers create apps on top of the [Aragon dapp](https://github.com/aragon/aragon) and [aragonOS](http://wiki.aragon.one/documentation/aragonOS/).
+Aragon is an open source, decentralized project fighting for the freedom to organize. We believe that decentralized organizations can solve the world's worst problems.
+
+We're looking for a Developer Relations to help third party developers create apps on top of the [Aragon dapp](https://github.com/aragon/aragon) and [aragonOS](http://wiki.aragon.one/documentation/aragonOS/). Our Dev Rels must be someone passionate in helping others understand technical subjects through technical writing, networking and educating people. That means that our ideal Dev Rels is not only a legit developer but also an Aragon Technical Evangelist.
 
 ### Responsibilities
 
-- Nurture the Aragon dev ecosystem
-- Attend and organize hackathons, helping devs get excited and started developing on Aragon.
-- Developer tutorials and documentation
-- Help external contributors
-- Find interesting projects that might be eligible for grants
+- Write technical documentation and tutorials for developers
+- Nurture and support the Aragon dev ecosystem
+- Attend and organize hackathons, helping devs get excited and started developing on Aragon
+- Support external contributors
+- Source, screen and select interesting projects that might be eligible for grants
 
 ### Requirements
 
+- Deep understanding of the crypto ecosystem and what's going on under the hood
 - Technical expertise and experience working professionally as a developer
 - Good social skills
 - Very good written communication skills
-- Interest in writing developer documentation
+- Experience writing technical documentation for developers
 - Interest in working with developers to help them use and build on top of Aragon
 - Being passionate about decentralization
-- Being an autonomous individual with a proactive attitude and a love for time self-management (We are focus-driven work, not hours-driven)
+- Being an autonomous individual with a proactive attitude and a love for time self-management
 
 ### Good to have
 
 - Experience contributing or managing open source projects
 - Experience creating documentation for developers, or documentation in general
+- Experience networking in blockchain events
 
 ### How to apply
 
 - Write an email with the following attachments plus any information you consider important to *hiring@aragon.one*, with the subject `Developer Relations opening application`.
 
-- Please send along your GitHub profile, LinkedIn, and references we can get in touch with.
+- Please send along your GitHub, LinkedIn/CV, Twitter/Reddit profiles, and references we can get in touch with.
+- Bonus: Why did Aragon caught your attention? Which challenge within the crypto space you find more interesting?
```

Comment:
Should this be `most interesting` or `more interesting than others`?

Reply:
I think it's the same! So,I'd say to keep it short.